pom.xml

```diff
@@ -128,6 +128,11 @@
       <artifactId>commons-logging</artifactId>
       <version>1.3.4</version>
     </dependency>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <version>1</version>
+    </dependency>
```

Comment:
It's unfortunate that javax didn't use a bom, like jakartaee does. I hate managing the javax dependency versions individually.